src/dashboard/apps/renewable/tests/test_view.py

```diff
@@ -148,3 +151,123 @@ def test_manage_views_get_context_data(rf, settings):
         renewable_dps, expected_renewable_dps, strict=True
     ):
         assert renewable.id == expected_renewable.id
+
+
+def _setup_bulk_update_renewables_test_data(user, entity_name, size):
+    """Fixture to setup bulk_update_renewables() test data."""
+    entity = EntityFactory(users=(user,), name=entity_name)
+
+    assert DeliveryPoint.objects.all().count() == 0
+    DeliveryPointFactory.create_batch(
+        size=size,
+        entity=entity,
+        has_renewable=True,
+        is_active=True,
+    )
+    assert DeliveryPoint.objects.all().count() == size
+
+    return entity.get_unsubmitted_quarterly_renewables()
```

Comment:
I don't see where it's used.

Reply:
It's probably because it's no longer used 😅
I will delete it!